packages/google-workspace-adapter/src/fix_elements/replace_groups_domain.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ *                      Copyright 2024 Salto Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FixElementsHandler } from '@salto-io/adapter-components'
+import { ChangeError, InstanceElement, isInstanceElement } from '@salto-io/adapter-api'
+import { collections } from '@salto-io/lowerdash'
+import { DEFAULT_PRIMARY_DOMAIN, UserConfig } from '../config'
+import { Options } from '../definitions/types'
+import { DOMAIN_TYPE_NAME, GROUP_TYPE_NAME } from '../constants'
+
+const { awu } = collections.asynciterable
+
+const replaceDomainInfo = (group: InstanceElement, domain: string): ChangeError => ({
+  elemID: group.elemID,
+  severity: 'Info',
+  message: `Replaced to domain of the ${group.value.name} group to ${domain}.`,
```

Comment:
this doesn't look right :) maybe 
```suggestion
  message: `Replaced the domain of group ${group.value.name} with ${domain}.`,
```
or
```suggestion
  message: `Replaced domain ${<domain>} of group ${group.value.name} with ${domain}.`,
```